docs/sources/mimir/references/architecture/components/ruler/index.md

```diff
@@ -12,19 +12,19 @@ weight: 130
 The ruler is an optional component that evaluates PromQL expressions defined in recording and alerting rules.
 Each tenant has a set of recording and alerting rules and can group those rules into namespaces.
 
+Evaluating rules generates new samples. Those samples are then passed to an in-process [distributor](../distributor) to be ingested and made available for further queries.
+Configuration of the built-in distributor uses [its respective configuration parameters](../../../../configure/configuration-parameters/#distributor).
+
 ## Operational modes
 
 The ruler supports two different rule evaluation modes:
 
 ### Internal
 
-This is the default mode. The ruler internally runs a querier and distributor, and evaluates recording and alerting rules in the ruler process itself.
+This is the default mode. The ruler internally runs a querier, and evaluates recording and alerting rules in the ruler process itself.
 To evaluate rules, the ruler connects directly to ingesters and store-gateways, and writes any resulting series to the ingesters.
 
-Configuration of the built-in querier and distributor uses their respective configuration parameters:
-
-- [Querier](../../../../configure/configuration-parameters/#querier)
-- [Distributor](../../../../configure/configuration-parameters/#distributor)
+Configuration of the built-in querier uses [its respective configuration parameters](../../../../configure/configuration-parameters/#querier).
```

Comment:
[Nit] I think respective is for referring to multiple components, not necessary when just referring to one.
```suggestion
Configuration of the built-in querier uses [its configuration parameters](../../../../configure/configuration-parameters/#querier).
```